adagrams/game.py

```diff
@@ -1,11 +1,91 @@
+import random
+import copy
+from collections import Counter
+
+LETTER_POOL = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+}
+
+
+score_table = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+}
+
+
 def draw_letters():
-    pass
+    random_list = []
+    for letter, frequency in LETTER_POOL.items():
+        random_list += list(letter * frequency)
+    return random.sample(random_list, 10)
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_copy = copy.deepcopy(letter_bank)
```

Comment:
Remember that the `letter_bank` is a list of letters (strings). Strings don't need to be copied, since they cannot be modifiied. All this code needs to do is remove letters from the copied list, which is just a change in membership of that list. So a shallow copy (copying the membership of the original list) is the only copy we need to use.